### PR TITLE
[ros2topic bw] Monotonic clock, units, fstring

### PR DESCRIPTION
**Pull request by sloretz**
This PR fixes a few issues I noticed while using `ros2topic bw`.

* Use `time.monotonic()` so time measurements aren't affected by system clock time jumps
* Fix units on printed message sizes - they're sizes not size per second
* Don't use any decimal places when printing bytes, continue using 2 decimal places with KB and MB
* Make the periodic print easier to understand (subjective)
  * It wasn't clear to me what `window:` was, so output now says it's a measurement considering that many messages
  * mean/min/max are sizes not bandwidth so output now includes word "size"
* Use `f` strings

What the current output in `eloquent` looks like:

```
$ ros2 topic bw /topic
Subscribed to [/topic]
average: 61.08B/s
	mean: 28.00B/s min: 28.00B/s max: 28.00B/s window: 2
average: 58.43B/s
	mean: 28.00B/s min: 28.00B/s max: 28.00B/s window: 4
average: 57.60B/s
	mean: 28.00B/s min: 28.00B/s max: 28.00B/s window: 6
average: 57.19B/s
	mean: 28.00B/s min: 28.00B/s max: 28.00B/s window: 8
```

What the new output looks like:
```
$ ros2 topic bw /topic
Subscribed to [/topic]
63 B/s from 2 messages
	Message size mean: 24 B min: 24 B max: 24 B
55 B/s from 4 messages
	Message size mean: 24 B min: 24 B max: 24 B
52 B/s from 6 messages
	Message size mean: 24 B min: 24 B max: 24 B
51 B/s from 8 messages
	Message size mean: 24 B min: 24 B max: 24 B
```

### CODE REVIEW

**Comment by sloretz** [comment]
CI (testing just ros2topic)
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/9466/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/5196/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/7739/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/9392/)
* Windows-container [Build Status](http://ci.ros2.org/job/ci_windows-container/232/)

**Comment by sloretz** [comment]
CI (since it's been a while, testing just ros2topic)
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/10258/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/5796/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/8339/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/10142/)

**Comment by sloretz** [comment]
CI (testing just `ros2topic`)
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/10265/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/5803/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/8346/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/10152/)

File: ros2topic/ros2topic/verb/bw.py [hunk 1]
```diff
@@ -58,6 +58,18 @@ def positive_int(string):
     return value
 
 
+def str_bytes(num_bytes):
+    return f'{num_bytes:.0f} B'
+
+
+def str_kilobytes(num_bytes):
+    return f'{num_bytes/1000:.2f} KB'
```
Comment (dirk-thomas) [inline ros2topic/ros2topic/verb/bw.py:66]: Shouldn't this be divided by 1024?

Same below.

Comment (sloretz) [reply ros2topic/ros2topic/verb/bw.py:66]: depends on one's opinion about kibi vs kilo

https://en.wikipedia.org/wiki/Kibibyte
https://en.wikipedia.org/wiki/Kilobyte

Comment (jacobperron) [reply ros2topic/ros2topic/verb/bw.py:66]: Internet searches suggest it's debatable :man_shrugging: 

If we divide by 1024, then we should probably also change the units to `KiB`.

Comment (hidmic) [reply ros2topic/ros2topic/verb/bw.py:66]: +1 to `KiB`

**Pull request merged**